frontend/components/transaction/BidTransaction.vue

```diff
@@ -52,22 +61,13 @@
                 !auctionTransaction.isActive ||
                 auctionTransaction.isFinished ||
                 !isWalletAuthorized ||
-                !isCollateralAuthorised ||
+                !isCollateralAuthorized ||
                 !isEnoughDeposited
```

Comment:
Please use newly created `isCorrect` values here instead of computed props. Please remove unused computed props, since now we want checking logic to be contained within panels and not reimplemented (introducing some inconsistencies) outside of the panels.